modules/110-istio/templates/kiali/configmap.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 data:
```

Comment:
Need to wipe extra privileges from rpac-for-us. I suppose, that we need to wipe all the clusterroles.